src/Common/TaskStatsInfoGetter.cpp

```diff
@@ -1,5 +1,6 @@
 #include "TaskStatsInfoGetter.h"
 #include <Common/Exception.h>
+#include "base/defines.h"
```

Comment:
style

Reply:
Yep, thanks. I've started to use vim+ycm+clangd (I was using libclang for a long time), which automatically insert headers, will fix this.